scripts/xtp_qmmm2qm.in

```diff
@@ -0,0 +1,73 @@
+#! /usr/bin/env python3
+#
+# Copyright 2009-2020 The VOTCA Development Team (http://www.votca.org)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import h5py
+import sys
+import os
+import argparse
+import importlib
+
+VERSION = '@PROJECT_VERSION@ #CSG_GIT_ID#'
+PROGTITLE = 'THE VOTCA::XTP QMMM orbfile converter'
+PROGDESCR = 'Convert QMMM orb file into a DFTGWBSE-like orb file usable by gencube,densityanalysis,...'
+VOTCAHEADER = '''\
+==================================================
+========   VOTCA (http://www.votca.org)   ========
+==================================================
+{progtitle}
+please submit bugs to @PROJECT_CONTACT@
+xtp_qmmm2qm, version {version}
+'''.format(version=VERSION, progtitle=PROGTITLE)
+
+
+parser = argparse.ArgumentParser(prog='xtp_qmmm2qm')
+
+parser.add_argument("-f", "--file", help="Input File (String)")
+parser.add_argument("-r", "--region", help="QMMM region number (Index)")
+parser.add_argument('-v', '--verbosity', action="count", help="Verbose mode")
+args = parser.parse_args()
+
+
+region = "region_" + args.region
+file_name, ext = os.path.splitext(args.file)
+
+
+print (VOTCAHEADER)
+print("...Input file: ", args.file)
+
+print("...Selected region: ", region)
+
+
+# Read input hdf5 file
+data_qmmm = h5py.File(args.file, 'r')
```

Comment:
This handler is never closed. Either add a `data_qmmm.close()` statement at the end or use a `with` statement like:
```py
with h5py.File(args.file, 'r') as data_qmmm:
       code here
```

Reply:
Good catch. I will add data_qmmm.close()

Reply:
actually the 'with' approach is better, because if something inside the 'with' block crashes the code the 'with' block will make sure that all file handles and stuff are properly closed.

Reply:
ok